tutorials/physics/rigid_body.rst

```diff
@@ -0,0 +1,51 @@
+.. _doc_rigid_body
+
+Rigid Body
+=========
+
+What is a rigid body?
+--------------------
+
+Rigid body is a rigid (non deformable) physical object that is directly controlled by physics engine in order to simulate phisical behaviour of objects.
+It's necessary to add one or more shapes that define the final shape of rigid body, you can set different position of these primitive shapes and this affect the COM (center of mass) of body.
+
+How to control rigid body
+--------------------------
+
+To alter the rigid body behaviour you can set some parameters like damping, friction, mass, bounce, gravity scale, etc..
+
+While to control the movement of the rigid body, you have many ways and you have to choose the correct one depending of the case.
+
+When you need to move a rigid body one time you can use all functions defined by *spatial node*, for example ``set_global_transform`` or ``look_at``. These functions are perfectly fine to create a teleport but must not be called each frame to move the rigid body because in this case the physics engine will not be able to handle it correctly and the physics simulation will not be correct.
+For example one common mistake is to use ``look_at`` each frame, this break physics similation. Few line below I'll show how to implement ``look_at`` correctly.
+
+The fact that you can't use ``set_global_transform`` or ``look_at`` functions doesn't mean that you can't have full control of rigid body, rather by adding forces, impulse, or by setting velocity, in the ``_integrate_forces(state)`` you will be able to achieve any movement you desire, even if it's not physically possible!
```

Comment:
The fact that you can't use ``set_global_transform()`` or ``look_at()`` methods doesn't mean that you can't have full control of a rigid body. Instead, you can control it by using the ``_integrate_forces()`` callback. In this function, you can add forces, apply, impulses, or set the velocity in order to achieve any movement you desire.